hwdb.d/70-mouse.hwdb

```diff
@@ -343,6 +343,10 @@ mouse:usb:v046dpc24c:name:Logitech G400s Optical Gaming Mouse:*
 mouse:usb:v046dpc07e:name:Logitech Gaming Mouse G402:*
  MOUSE_DPI=400@1000 *800@1000 1600@1000 3200@1000
 
+# Logitech Logitech G403 Prodigy (Wired)
```

Comment:
why the duplicate 'Logitech' here? looks good otherwise, thanks

Reply:
That was an unfortunate typo!

Reply:
Fixed the typo.

Sorry for the unecessary noise.

Reply:
thanks, all good now